frontend/uddhar-frontend/src/volunteer/components/OrganizationList.jsx

```
@@ -81,32 +115,8 @@ function OrganizationList({ organizations, handleNext }) {
             className="flex gap-6 overflow-x-hidden scroll-smooth px-4 py-4"
             onClick={() => handleNext}
           >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)